google/datalab/bigquery/commands/_bigquery.py

```diff
@@ -719,12 +719,14 @@ def _table_cell(args, cell_body):
       if args['project'] is None:
         datasets = [google.datalab.bigquery.Dataset(args['dataset'])]
       else:
-        datasets = [google.datalab.bigquery.Dataset((args['project'], args['dataset']))]
+        context = google.datalab.Context(args['project'],
```

Comment:
Oh nice catch! It's interesting I also made the same mistake in the test, and wrote it to expect project twice! :)